src/options.c

```diff
@@ -563,30 +585,31 @@ void optionsParseDisplay(char *optarg)
 void optionsParseThumbnail(char *optarg)
```

Comment:
It's broken, this should work like `master`: `scrot -t 20`
Wrong result: `scrot: option --thumb: resolution height is missing`

Reply:
Now there is another case, it does not generate the thumbnail image file with: `scrot -t 100x100`

Reply:
```diff
             errx(EXIT_FAILURE, "option --thumb: resolution height '%s' is %s",
                 height, errmsg);
         }
+
+        // allow create thumbnail
+        opt.thumb = 1;
+
     } else {
         /* optarg is a percentage. */
         opt.thumb = options_parsenum(optarg, 1, INT_MAX, &errmsg);
```